.github/workflows/helm-chart-release.yaml

```
@@ -68,7 +68,24 @@ jobs:
           helm repo add grafana https://grafana.github.io/helm-charts
           helm dependency update
 
+      - name: Create temporary CHANGELOG file for helm release
+        run: |
+          cd charts/irs-helm
```

it is good practice to use `working-directory: charts/irs-helm/` instead of cd 
```suggestion
      - name: Create temporary CHANGELOG file for helm release
        working-directory: charts/irs-helm/
        run: |
```